db/migrations/20220413104717_goteo_new_faq.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+use Goteo\Model\Faq;
+
+/**
+ * Migration Task class.
+ */
+class GoteoNewFaq
+{
+  public function preUp()
+  {
+      // add the pre-migration code here
+  }
+
+  public function postUp()
+  {
+      // add the faq-migration code here
+      // Create default slugs for everybody
+      foreach(Faq::query("SELECT id,title FROM faq")->fetchAll(\PDO::FETCH_OBJ) as $faq) {
+        $slug = Faq::idealiza($faq->title, false, false, 150);
+        try {
+            // If duplicate, let it null
+            Faq::query("UPDATE faq SET slug=:slug WHERE id=:id", [':id' => $faq->id, ':slug' => $slug]);
+        } catch(\PDOException $e) {
+            //
+        }
+      }
+  }
+
+  public function preDown()
+  {
+      // add the pre-migration code here
+  }
+
+  public function postDown()
+  {
+      // add the post-migration code here
+  }
+
+  /**
+   * Return the SQL statements for the Up migration
+   *
+   * @return string The SQL string to execute for the Up migration.
+   */
+  public function getUpSQL()
+  {
+     return "
+
+      CREATE TABLE `faq_section` (
+            `id` BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT,
```

Comment:
Are we really going to have so many FAQ sections that we need a BIGINT for the ID?

Reply:
No, we won't. You are right